seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -34,7 +34,7 @@
       "title": "Say Hello to HTML Elements",
       "description": [
         "Welcome to freeCodeCamp's coding challenges. These will walk you through web development step-by-step.",
-        "We recommend you do these in order, but feel free to skip around by clicking the map on the left.",
+        "We recommend you do these in order, but feel free to skip around by clicking the map in the navigation bar.",
```

Comment:
Thanks for the PR @jdeprey! Like the issue, I prefer `menu` over `navigation bar`, I think `menu` is more widely understood.